ansible-ipi-install/roles/node-prep/tasks/10_validation.yml

```diff
@@ -157,6 +157,9 @@
     status_code: [-1,200]
     timeout: 1
   register: check_url
+  until: _result.status == 200
```

Comment:
not sure but shouldn't this be check_url.status?

Reply:
Should be changed in all items :) (thanks!)